docs/guides/tutorials/cloudflare_workers.rst

```diff
@@ -0,0 +1,261 @@
+to create a new instance:
+
+.. code-block:: bash
+
+    $ edgedb project init
+
+It creates an :code:`edgedb.toml` config file, and a schema file 
+:code:`dbschema/default.esdl`.
+
+It also spuns up an EdgeDB instance and “links” it to the current directory. 
+As long as you’re inside the project directory, all CLI commands will 
+be executed against this instance.
+
+You can run :code:`edgedb` in your terminal to open an 
+interactive REPL to your instance. 
+
+.. code-block:: bash
+
+    $ edgedb
+
+**Install The EdgeDB NPM package**
+
+.. code-block:: bash
+
+    $ npm install edgedb # or pnpm, yarn, bun
+
+**Extend The Default Schema (Optional)**
+
+You can extend the default schema, :code:`dbschema/default.esdl`, to define 
+your data model, and then try it out in the Cloudflare Worker code.
+
+Add new types to the schema file:
+
+.. code-block:: sdl
+
+    module default {
+      type Movie {
+        required property title -> str {
+          constraint exclusive;
+        };
+        multi link actors -> Person;
+      }
+
+      type Person {
+        required property name -> str;
+      }
+    }
+
+Then apply the schema schema to your EdgeDB instance:
+
+.. code-block:: bash
+
+    $ edgedb migration create
+    $ edgedb migrate
+
+Using EdgeDB in a Cloudflare Worker
+====================================
+
+Open the :code:`index.ts` file from the :code:`src` directory in your project, 
+and remove the default code.
+
+To interact with your **local EdgeDB instance**, use the following code:
+
+.. code-block:: typescript
+
+    import * as edgedb from 'edgedb';
+
+    export default {
+     async fetch(
+      _request: Request, 
+      env: Env, 
+      ctx: ExecutionContext
+     ): Promise<Response> {
+      
+      const client = edgedb.createHttpClient({
+       tlsSecurity: 'insecure',
+       dsn: '<your-edgedb-dsn>',
+      });
+
+      const movies = await client.query(`select Movie { title }`);
+
+      return new Response(JSON.stringify(movies, null, 2), {
+       headers: {
+        'content-type': 'application/json;charset=UTF-8',
+       },
+      });
+     },
+    } satisfies ExportedHandler<Env>;
+
+
+.. note:: EdgeDB DSN
+
+    Replace :code:`<your-edgedb-dsn>` with your EdgeDB DSN.
+    You can obtain your EdgeDB DSN from the command line by running:
+
+    .. code-block:: bash
+
+        $ edgedb instance credentials --insecure-dsn
+
+.. note:: tlsSecurity
+
+    The :code:`tlsSecurity` option is set to :code:`insecure` to allow 
+    connections to a local EdgeDB instance. This lets you test your
+    Cloudflare Worker locally. **Don't use this option in production.**
+
+**Client Setup with EdgeDB Cloud**
+
+If you're using EdgeDB Cloud, you can instead use the following code to 
+set up the client:
+
+.. code-block:: typescript
+
+   const client = edgedb.createHttpClient({
+   	instanceName: env.EDGEDB_INSTANCE,
+   	secretKey: env.EDGEDB_SECRET_KEY,
+   });
```

Comment:
Minor nit:

```suggestion
   const client = edgedb.createHttpClient({
     instanceName: env.EDGEDB_INSTANCE,
     secretKey: env.EDGEDB_SECRET_KEY,
   });
```

In general it'd be good to have all of this TS code formatted with the default Prettier config.